for_streamlit_only/manual.py

```diff
@@ -16,15 +22,28 @@ def example_data_html(spectrometer):
 
 
 def show_manual():
-    st.markdown(f'''
-            <img src="https://sersitive.eu/wp-content/uploads/LOGO.png"
-            style="
-            margin: auto;
-            width: 80%;
-            padding:0px 6px 45px 25%; 20px;
-            "/>''',
-                unsafe_allow_html=True
-                )
+    """
+    Shows manual on front page if data is not uploaded
+    """
+    
+    # company logo
+    with open('logos/logo.png', 'rb') as f:
+        data = f.read()
+    
+    bin_str = base64.b64encode(data).decode()
+    html_code = f'''
+                    <img src="data:image/png;base64,{bin_str}"
+                    style="
+                         margin: auto;
+                         margin-top:-30px;
+                         width: 65%;
+                         padding:0px 6px 20px 25%;
+                         "/>
+                '''
```

Comment:
To warto by wyrzucić do osobnej funkcji, zwracać ten html_code i dać nad tym @st.cache